Develop

mampacch opened the pull request.

aksshriv1 commented on src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_1320/DefectReopenChangeUnit.java:
```diff
@@ -0,0 +1,162 @@
+ *
+ * @author shunaray
+ */
+@ChangeUnit(id = "r_defect_reopen_quality", order = "013204", author = "shunaray", systemVersion = "13.2.0")
```
update order to 013205

gurdeep-singh-49 committed:
added changes for Role : Project Admin
Changes : Project admin now change add/update/list all the user under that project

gurdeep-singh-49 committed:
changes for return ProectAdmin bean, if person is having multiple roles.

gurdeep-singh-49 committed:
added new test cases and updated few

gurdeep-singh-49 committed:
Resolving sonar issue.

gurdeep-singh-49 committed:
adding more test case to cover sonar coverage.

gurdeep-singh-49 committed:
adding more test case to cover sonar coverage.

gurdeep-singh-49 committed:
change as per sonar suggestion.

aksshriv1 committed:
Explore Defect Fix for KPI Test Execution Time

aksshriv1 committed:
Merge pull request #18 from PublicisSapient/bugFix/DTS-49493_TestExecutionKPI_AS

Explore Defect Fix for KPI Test Execution Time

gurdeep-singh-49 committed:
Merge branch 'develop' into feature/DTS-48665_ProjectAdminAccessChanges

gurdeep-singh-49 committed:
Merge branch 'develop' into qa-master-sync

gurdeep-singh-49 committed:
Merge pull request #21 from PublicisSapient/qa-master-sync

Qa master sync

gurdeep-singh-49 committed:
Merge branch 'develop' into feature/DTS-48665_ProjectAdminAccessChanges

gurdeep-singh-49 committed:
adding execution part as well

gurdeep-singh-49 committed:
git comment

gurdeep-singh-49 committed:
git comment

gurdeep-singh-49 committed:
commenting execution as per discussion

gurdeep-singh-49 committed:
Merge branch 'develop' into feature/DTS-49477-eclipse-formatter-spotless

gurdeep-singh-49 committed:
git comment

gurdeep-singh-49 committed:
added copyright

gurdeep-singh-49 committed:
removing commented part as sonar is not allowing comments

gurdeep-singh-49 committed:
Merge branch 'feature/DTS-49477-eclipse-formatter-spotless' of https://github.com/PublicisSapient/knowhow-api into feature/DTS-49477-eclipse-formatter-spotless

gurdeep-singh-49 committed:
Merge pull request #15 from PublicisSapient/feature/DTS-48665_ProjectAdminAccessChanges

Feature/dts 48665 project admin access changes

gurdeep-singh-49 committed:
Merge branch 'develop' into feature/DTS-49477-eclipse-formatter-spotless

gurdeep-singh-49 committed:
Merge pull request #23 from PublicisSapient/feature/DTS-49477-eclipse-formatter-spotless

adding execution part as well
